Formula/sonarqube-lts.rb

```diff
@@ -52,5 +55,27 @@ def plist
 
   test do
     assert_match "SonarQube", shell_output("#{bin}/sonar status", 1)
+
+    begin
+      pid = fork do
+        exec bin/"sonar", "start"
+      end
+
+      puts "pid: #{pid}"
+
+      seconds = 0
+      loop do
+        status = shell_output("#{bin}/sonar status")
+        break if status.match?("is running") || (seconds += 5) > 60
+
+        sleep 5
+      end
```

Comment:
Now that this is working we can just wait like other formula tests, no need for complicated loops.
```suggestion
      puts "pid: #{pid}"
      sleep 60
```